IdentityCore/src/cache/crypto/MSIDAssymetricKeyPair.m

```diff
@@ -116,7 +116,7 @@ - (NSString *)keyModulus
         iterator++; // TYPE - bit stream mod
         int mod_size = [self derEncodingGetSizeFrom:publicKeyBits at:&iterator];
         NSData *subData=[publicKeyBits subdataWithRange:NSMakeRange(iterator, mod_size)];
-        _keyModulus = [[subData subdataWithRange:NSMakeRange(1, subData.length-1)] base64EncodedStringWithOptions:0];
+        _keyModulus = [[subData subdataWithRange:NSMakeRange(1, subData.length-1)] msidBase64UrlEncodedString];
```

Comment:
Do you want to change everything from base64 to base64URL just to be safe? (e.g keyExponent param)

Reply:
good point, added